.envrc

```diff
@@ -90,12 +83,7 @@ if ! type source_up >/dev/null 2>/dev/null; then
 fi
 
 # The fissile stemcell image that is used as the base
-if [ "${USE_SLE_BASE}" = "false" ]
-then
-    export FISSILE_STEMCELL=${FISSILE_STEMCELL:-${FISSILE_DOCKER_REGISTRY}${FISSILE_DOCKER_ORGANIZATION}/fissile-stemcell-opensuse:$FISSILE_STEMCELL_VERSION}
-else
-    export FISSILE_STEMCELL=${FISSILE_STEMCELL:-${FISSILE_DOCKER_REGISTRY}${FISSILE_DOCKER_ORGANIZATION}/fissile-stemcell-sle12:$FISSILE_STEMCELL_VERSION}
-fi
+export FISSILE_STEMCELL=${FISSILE_STEMCELL:-${FISSILE_DOCKER_REGISTRY}${FISSILE_DOCKER_ORGANIZATION}/fissile-stemcell-sle12:$FISSILE_STEMCELL_VERSION}
```

Comment:
Could you pull `${FISSILE_DOCKER_REGISTRY}${FISSILE_DOCKER_ORGANIZATION}/fissile-stemcell-sle12:$FISSILE_STEMCELL_VERSION` into a separate variable? Maybe `FISSILE_DEFAULT_STEMCELL`. It doesn't need to be exported. It will ease the reading.